api_compatibility_policy.md

```diff
@@ -125,14 +124,12 @@ See the current list of [alpha features](https://github.com/tektoncd/pipeline/bl
   i.e. by providing a 9 month support period.
 
 - Beta features are reviewed for promotion to GA/Stable on a regular basis. However, there is no guarantee that they will be promoted to GA/stable.
-
-- For beta API versions, beta is the highest level of stability possible for any feature.
   
 ### GA/Stable features
 
-- GA/Stable features are present in a [GA CRD](#ga-crds) only.
+- GA/Stable features are enabled by default.
 
-- GA/Stable features are enabled by default
+- GA/Stable features are features that have been promoted from beta to the highest level of stability. They cannot be disabled in any CRD version.
```

Comment:
These two sentences
> GA/Stable features are enabled by default.

and 

> GA/Stable features are features that have been promoted from beta to the highest level of stability. They cannot be disabled in any CRD version.

kind of contradict each other. "Enabled by default" implies that they can be disabled, but the default is "enabled".

I would say that stable features are not controlled by the `enable-api-fields` flag anymore.
I guess that in this context we are only talking about feature flags for API-driven features - other features may still be controlled by a separate dedicated feature flag even once they're stable.